AppController/lib/haproxy.rb

```diff
@@ -407,12 +410,15 @@ def self.get_haproxy_stats(app_name)
 
       if service_name == "BACKEND"
         total_req_in_queue = parsed_info[REQ_IN_QUEUE_INDEX].to_i
+        current_sessions = parsed_info[CURRENT_SESSIONS_INDEX].to_i
         Djinn.log_debug("#{full_app_name} #{service_name} Queued Currently " +
           "#{total_req_in_queue}")
+        Djinn.log_debug("Current sessions for #{full_app_name} per AppServer " +
```

Comment:
Are you sure that `scur` is a number of sessions per AppServer? I would expect that for backend items in haproxy stats this number is total number of sessions for the backend. When for server items it is number of sessions for the specific server (AppServer in this case).